pandas/tests/io/json/test_normalize.py

```diff
@@ -287,6 +287,30 @@ def test_shallow_nested(self):
         expected = DataFrame(ex_data, columns=result.columns)
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.skipif(not PY36, reason="drop support for 3.5 soon")
+    def test_nested_meta_path_with_nested_record_path(self, state_data):
+        # GH 27220
+        result = json_normalize(
+            data=state_data,
+            record_path=["counties", "name"],
+            meta=["state", "shortname", ["info", "governor"]],
+            errors="ignore",
+        )
+        ex_data = {
```

Comment:
Instead of using a dict can you construct this with literal values? If you use a list of lists can circumvent 3.5 ordering issues with a dict

If you can simplify expectation would help a lot as well so as not to write out the same values 21 times

Reply:
Thanks for the suggestion. It is indeed better. I simply copied other test cases. I will change accordingly.